ultra/ultra/scenarios/common/distributions.py

```diff
@@ -775,7 +775,24 @@
             },
             "west-east": None,
             "west-north": None,  # blocking
-            "west-south": None,
+            "west-south": {
+                "vehicles": 100,
+                "start_end_on_different_lanes_probability": 0.0,
+                "distribution": {
+                    "default": 0.2,
+                    "aggressive": 0.8,
+                },
+                "begin_time_init": {
+                    "func": burst_begin_time_init_func,
+                    "params": {
+                        "vehicle_cluster_size": (1, 1),
+                        "time_between_cluster": (20, 30),
+                        "time_for_each_cluster": 1,
+                    },
+                },
+                "has_turn": True,
+                "deadlock_optimization": True,
+            },
             "east-west": None,
```

Comment:
Do you know if it's still possible for a continuous control baseline to learn with these extra social vehicles?

Reply:
I had some preliminary training done with sac, but it was not much, however I will run a bit longer experiment with the older reward model and then with the new one. I will convert this PR back to a draft and once completed I will let you know 👍